tests/checkers/unittest_deprecated.py

```diff
@@ -57,6 +57,31 @@ def deprecated_decorators(self) -> set[str]:
 class TestDeprecatedChecker(CheckerTestCase):
     CHECKER_CLASS = _DeprecatedChecker
 
+    def test_deprecated_attribute(self) -> None:
+        # Tests detecting deprecated attribute
+        node = astroid.extract_node(
+            """
+        class DeprecatedClass:
+            deprecated_attribute = 42
```

Comment:
This won't pass unless you add patch `deprecate_attribute` into the dict of deprecated attributes. But I like the idea of doing that, since the test won't frequently go out of date as attributes are removed in future python versions.

Reply:
So should I add or not ?

Reply:
Sorry, I didn't look at the top of this file. The top of this file (`_DeprecatedChecker`) is where they are patched, so you should add a `deprecated_attributes()` method.

Reply:
Should I publish a PR with the changes and you can review it ?

Reply:
No, we should continue editing this PR. I'm about to push more commits to it.